core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -144,6 +144,9 @@
    */
   private volatile FileSystemContextReinitializer mReinitializer;
 
+  /** Whether to disable URI scheme validation for file systems using this context.  */
+  private boolean mDisableUriValidation = false;
```

Comment:
nit: typically we use `enableUriValidation` or `uriValidationEnabled` rather than `disableUriValidation` in this codebase, to simplify the logic slightly.

Reply:
b87bceb301b8afec00dee8cdc6872ca8b486ef65